pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -264,6 +291,12 @@ func (c *InstallerController) manageInstallationPods(operatorSpec *operatorv1.Op
 		break
 	}
 
+	statusConfigMap.Data["phase"] = phaseStatus
```

Comment:
I expected to see a helper for updating status for a revision that interpreted the status update and decided what the phase should be .

Reply:
We want to look at the status update for each node and set phase to `failed` if any nodes are failed, otherwise set it as `success`, right? If that's the approach then can't I just interpret it here while we're already working on each node's status update?